microsoftgraph/microsoft-graph-explorer-v4
# Fix: ProfileType persistence

## Overview

FIxes #1073 
Refactors the profile component to ensure requests are outside the component
Moves profileType into the profile state property
Chains calls to the profile endpoints

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* Log in to GE
* Open dev tools
* Look at the profileType in the profile state
* Logout 
* Notice the profile is cleared
